### DISCUSSION TITLE
Request: add a written quick start

### DISCUSSION THREAD

**Discussion by xpe** [rustic-rs/rustic#797] [General]
Video quick starts may be hip, but written ones are preferable for these reasons:

1. people read at different speeds (watching a video is painfully slow for me)
2. it isn't practically feasible to skim a video
3. copy-paste (I wanted to copy-paste the example config file)
4. written docs are easier to edit and therefore maintain.

Thanks for this project. FWIW, my criticisms about video walkthroughs are universal and not particular to this wonderful project! 👍

**Comment by xpe** [comment]
~Ah, [the crates.io documentation about rustic-backup](https://crates.io/crates/rustic-backup) is what I am looking for. Would you be amenable to moving it into the source code?~

Drat. That is a *different* project.

**Comment by simonsan** [comment]
Closing in favour of https://github.com/rustic-rs/rustic/issues/799

**Reply by xpe** [reply]
Thanks for 'promoting' this to be an issue rather than just a discussion.

**Reply by simonsan** [reply]
Do you feel like working on this issue and writing a quick start/introduction? That being said, I think it's valuable to keep the video quick start, but in addition just have a documentation part for it where people could also read it as well.

**Comment by simonsan** [comment]
On topic:


1. people read at different speeds (watching a video is painfully slow for me)
1. it isn't practically feasible to skim a video
1. copy-paste (I wanted to copy-paste the example config file)
1. written docs are easier to edit and therefore maintain.

Answers:

1. that's understandable, and I think a straight-out written quick start/introduction to how to work with rustic is a reasonable thing to have -> in addition.
2. same as 1.
3. same as 1.
4. we're using [autocast](https://github.com/k9withabone/autocast) to automate most of it with a yaml file that generates the actions: https://github.com/rustic-rs/rustic/blob/main/docs/screenshots/gettingstarted.yaml so it indeed is easy to maintain.

**Reply by xpe** [reply]
Thanks for the detailed answer. Glad to learn about autocast as well. I stand corrected.
